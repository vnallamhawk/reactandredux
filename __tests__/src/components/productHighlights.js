import React from 'react';
import {configure} from 'enzyme';
import ShallowRenderer from 'react-test-renderer/shallow';
import ProductHighlights from '../../../src/components/productHighlights';
import {shallow,mount} from 'enzyme';
import Adapter from 'enzyme-adapter-react-15';
configure({adapter:new Adapter});

describe('Product Highlights Test -- Snapshot',()=>{
    it('capturing snapshot of Product Hightlight',()=>{
        const renderer = new ShallowRenderer();
        const result = renderer.render(<ProductHighlights/>)
        expect(result).toMatchSnapshot();
    })
})


describe('Testing of features rendering',()=>{
    it('capturing snapshot of Product Hightlight',()=>{
        let container,result;
        let highlights=["<strong>Wattage Output:</strong> 1100 Watts", "<strong>Number of Speeds:</strong> 3 "]
        container = shallow(<ProductHighlights highlights = {highlights}/>)
        result = container.find('.features').length;
        expect(result).toEqual(2);
    })
})
