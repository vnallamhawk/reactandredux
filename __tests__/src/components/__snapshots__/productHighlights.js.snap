// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Product Highlights Test -- Snapshot capturing snapshot of Product Hightlight 1`] = `
<div>
  <h2>
     Product highlights
  </h2>
  <ul>
    
  </ul>
</div>
`;
